deploy-board/deploy_board/templates/clusters/base_images_events.tmpl

```diff
@@ -30,4 +42,70 @@
         </tr>
         {% endfor %}
     </table>
-</div>
\ No newline at end of file
+    <div class="panel-footer clearfix">
+        <div class="pull-left">
+            <button id="rollbackItButton" type="button" class="btn btn-primary" title="Rollback to the picked From Image">Rollback to the selected From Image</button>
```

Comment:
Nit: picked -> selected. btw how about "Rollback to From Image of the selected event"

Reply:
ok